### PR TITLE
Clean up some dependencies.  Update to latest Hangfire.

### PR DESCRIPTION
**Pull request by adamhathcock**
Thanks for this library.  Just tidying up dependencies.

### CODE REVIEW

**Comment by marcoCasamento** [comment]
Hi, thanks to for the PR, nice, just a little thing: I'm trying to keep dependencies aligned with Hangfire.Core, so I'd avoid to upgrade Newtonsoft.Json to 9.0 for the net451 framework. 
If you can resubmit the PR I can merge it immediately, I'll otherwise edit & merge it in the weekend.
Thanks again

**Comment by adamhathcock** [comment]
You're right.  Hangfire.Core is targetting 5.0.0.  Weird.

Anyway, now reverted!

File: Hangfire.Redis.Tests/project.json [hunk 1]
```diff
@@ -1,29 +1,23 @@
...
+    },
 
-  "frameworks": {
-    "netcoreapp1.0": {
-      "dependencies": {
-        "Microsoft.NETCore.App": {
-          "type": "platform",
-          "version": "1.0.0"
+    "frameworks": {
+        "netcoreapp1.1": {
```
Comment (nj) [inline Hangfire.Redis.Tests/project.json]: `appveyor.yml` needs to reflect this as well, in order to build - got broken due to this.

Might be better to test against both versions.

File: Hangfire.Redis.StackExchange/project.json [hunk 2]
```diff
@@ -1,23 +1,14 @@
 ﻿{
-  "dependencies": {
-    "NETStandard.Library": "1.6.0",
-    "Hangfire.Core": "1.6.5",
-    "StackExchange.Redis": "1.1.608"
-  },
+    "dependencies": {
```
Comment (nj) [inline Hangfire.Redis.StackExchange/project.json]: Tip: Best to match existing indentation, giving less visual changes in diffs (and hence PR)

**Comment by marcoCasamento** [comment]
PR cleaning today, it seems that this one has become obsolete due to my lazyness (busyness) but have been anyway implemented in later commits. I'm closing it, sorry for that

**Pull request closed without merging**